lib/ansible/module_utils/facts/virtual/linux.py

```diff
@@ -104,6 +104,11 @@ def get_virtual_facts(self):
                 virtual_facts['virtualization_role'] = 'guest'
                 found_virt = True
 
+        # ensure 'container' guest_tech is appropriately set
+        if 'container' not in guest_tech:
```

Comment:
This shouldn't be necessary, since it's a set. It won't get added twice no matter what.

Reply:
good point - let me update